src/components/shared/Button/PaneToggle.js

```diff
@@ -1,19 +1,21 @@
 // @flow
 
-import { DOM as dom, PropTypes, Component } from "react";
+import { DOM as dom, Component } from "react";
 import classnames from "classnames";
 import Svg from "../Svg";
 import "./PaneToggle.css";
 
-type NextProps = {
+type Props = {
   collapsed: boolean,
-  handleClick: () => any,
+  handleClick: (string, boolean) => any,
   horizontal?: boolean,
   position: string
 };
 
 class PaneToggleButton extends Component {
-  shouldComponentUpdate(nextProps: NextProps) {
+  props: Props;
+
```

Comment:
Yes we could have props here but we also need to use it in below line

```js
shouldComponentUpdate(nextProps: Props)  {
```
I used the `type Props` to avoid duplication